src/app/clusters/unit-localization-server/unit-localization-server.cpp

```diff
@@ -68,6 +69,7 @@ CHIP_ERROR UnitLocalizationServer::SetSupportedTemperatureUnits(DataModel::List<
         mUnitsBuffer[i++] = item;
     }
     mSupportedTemperatureUnits = DataModel::List<TempUnitEnum>(mUnitsBuffer, units.size());
+    MatterReportingAttributeChangeCallback(kRootEndpointId, UnitLocalization::Id, SupportedTemperatureUnits::Id);
```

Comment:
This cannot be reported without bumping ConfigurationVersion. The code must be added to do the bump.

@ReneJosefsen 

Really, the solution is to only call `SetSupportedTemperatureUnits` to the same value over time unless the values actually change and the ConfigurationVersion is bumped.

When do you think this would be called by a product? All the time or only at startup and not change significantly at runtime?

Reply:
Removing this for now.